server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/SetMessagesMethodTest.java

```diff
@@ -1396,6 +1396,40 @@ public void setMessageShouldRequireDraftInOnlyDraftMailbox() {
     }
 
     @Test
+    public void setMessageShouldBeCompatibleWithIsDraftProperty() {
```

Comment:
what does it mean ?

Reply:
I propose " check INBOX will be able to save a message with isDraft,"
I'm kidding but "should be compatible" doesn't say what you explained.
So `setMessageShouldAllowDraftCreationWhenUsingIsDraftProperty` ?